robot/robot.md

```diff
@@ -0,0 +1 @@
+500年后醒来，他变成了HALO中的超级战士，需要帮助人类拯救地球。
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.